test/guardian_phoenix/socket_test.exs

```diff
@@ -3,25 +3,24 @@ defmodule Guardian.Phoenix.SocketTest do
   use ExUnit.Case, async: true
   use Phoenix.ChannelTest
 
-  @endpoint Guardian.Phoenix.SocketTest.Endpoint
+  alias Guardian.Phoenix.TestSupport.{MySocket, Impl, Endpoint}
 
+  @endpoint Endpoint
```

Comment:
It's actually more characters to use this module attribute in place of the module, let's ditch this?

Reply:
@doomspork Doing refactoring was hard to figure out why modules were failing because we did `__MODULE__.KEEP_GOING_WITH_THE_NAME`

Better to be explicit and simply to follow where the code is coming from.